traits/tests/test_constant.py

```diff
@@ -0,0 +1,54 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+
+import unittest
+
+from traits.api import Constant, HasTraits, TraitError
+
+
+class TestConstantTrait(unittest.TestCase):
+
+    def test_initial_value(self):
+        class TestClass(HasTraits):
+            c_atr = Constant(5)
+
+        self.assertEqual(5, TestClass().c_atr)
```

Comment:
Nitpick, but let's switch the order of arguments on all these `self.assertEqual` calls: the usual order in Python-land is `self.assertEqual(actual, expected)` rather than `self.assertEqual(expected, actual)`.

Reply:
I thought it was the other way around, because when I have : 
`self.assertEqual(4, TestClass().c_atr)`
I get the following error message (in PyCharm).
```
FAILED (failures=1)


5 != 4

Expected :4
Actual   :5
<Click to see difference>
```

Reply:
Hmm; interesting. It's not clear cut, and some languages make other decisions, but for Traits, let's be consistent with the existing code and with core Python style (and the examples in the unittest documentation) and use (actual, expected). It makes code harder to read if we use a mix of the two styles.

Reply:
(If you want some fun historical reading, including Guido arguing with himself, see https://mail.python.org/pipermail/python-dev/2010-December/106875.html)